content/Certification/Guides/PICSandPIXIT.md

```diff
@@ -0,0 +1,201 @@
+from the server list on an endpoint, feature maps, attribute and command lists
+correspond directly to the PICS.S.Fxx, PICS.S.Axxxx and PICS.S.Cxx.Xx PICS
+codes. The Matter SDK provides a tool to pre-fill these values in the PICS XML
+files so they do not have to be individually, manually filled in the PICS tool.
+
+-   [PICS Generator](https://github.com/project-chip/connectedhomeip/tree/master/src/tools/PICS-generator)
+
+PICS codes filled using this tool still need to be validated by the PICS tool.
+Note that due to device limitations, the tool will NOT fill the following
+categories of PICS codes, and these will need to be filled manually in the PICS
+tool:
+
+-   Event PICS (PICS.S.Exx)
+-   Client PICS
+-   MCORE (base.xml) PICS
+-   Manual or other non-element PICS
+-   PICS describing whether optionally-writable attributes are writable
+-   any other non-element PICS
+
+It is important to note that this script is NOT the official tool for PICS
+generation, just a helper to assist with this manual process. It is very
+important to go back and check that the values are as expected and to fill in
+the other PICS.
+
+### Verifying PICS using the IDM-10.4 certification test
+
+While not all PICS are verifiable on the device, we do have tests that verify
+the declared PICS against the device. This is one of the first tests that should
+be run at certification, as the PICS files are what determine which set of tests
+are required.
+
+To run these tests locally, follow the instructions at
+[Running Python tests locally](./python.md#running-tests-locally). The PICS
+checker test is TC-IDM-10.4, implemented in
+[TC_pics_checker.py](https://github.com/project-chip/connectedhomeip/blob/master/src/python_testing/TC_pics_checker.py).
+These tests run a single set of PICS XML files against an endpoint. The
+`--endpoint` and `--PICS` flags are therefore required.
+
+Note that you can run tests locally against the PICS XMLs for an endpoint by
+supplying the name of the directory containing the set of PICS XML files for
+that endpoint.
+
+
+## Setting PIXITs for Matter devices
+
+Matter tests do not currently have support to read PIXIT values from the PICS
+XML files. Instead, tests implement PIXITs as test-specific flags. When running
+locally, these are specified on the command line. When running in the test
+harness, these are specified in the test parameters section of the test
+configuration.
+
+## PICS for test selection
+
+The official source that the CSA certification team uses to determine if all the
+required tests have been run at certification is the submitted set of PICS XML
+files and the PICS tool.
+
+To generate this set of tests for each endpoint, load all the filled PICS XML
+files for a single endpoint into the PICS tool and validate the PICS files. The
+PICS files should validate properly. This will also generate the list of test
+cases.
+
+Note that the TH will also guide test selection by pre-selecting the required
```

Comment:
I assume TH = Test Harness, can we use TH (Test Harness)?

Reply:
Changed to "test harness"